docs/UserGuide.md

```diff
@@ -155,7 +155,7 @@ Example of usage:
 Expected Outcome:
 
 ```
-Nice, added a 500.0 µF
+Nice, added a 500.0 µF**
```

Comment:
Why are the stars added?

Reply:
Must have been by accident, I'll remove it now